kafka-connect-neo4j/src/test/kotlin/streams/kafka/connect/sink/Neo4jSinkTaskTest.kt

```diff
@@ -1351,6 +1353,54 @@ class Neo4jSinkTaskTest {
         }
     }
 
+    @Test
+    fun `should successfully parse nested timestamps`() {
```

Comment:
is this just timestamps, or more general sub-maps?

Reply:
is general sub-maps